src/ensembl/src/shared/components/question-button/QuestionButton.scss

```diff
@@ -17,6 +17,8 @@
   width: 16px;
   height: 16px;
   padding: 0;
+  height: 14px;
```

Comment:
Looks like you didnt remove/overwrite, the width and height above. width and height are specified twice :)

Reply:
sorry @jyothishnt just realised that the default height and width is 14px so we dont need to specify the width and height here.  do you agree?

Reply:
sorry ignore the above comment, i was looking at the wrong question icon but i notice that you can specify just the height or the width to 14px and it works, you dont have to specify both but probably best to specify both to be safe.